kubernetes/config/samples/etcd-bitnami.yaml

```diff
@@ -0,0 +1,41 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: etcd-service
+spec:
+  ports:
+  - name: etcd-client-port
+    port: 2379
+    protocol: TCP
+    targetPort: 2379
+  selector:
+    app: etcd
+
+---
+apiVersion: v1
+kind: Pod
+metadata:
+  labels:
+    app: etcd
+  name: etcd
+spec:
+  containers:
+  - image: bitnami/etcd:latest
+    name: etcd
+    env:
+    - name: ETCD_ENABLE_V2
+      value: "true"
+    - name: ALLOW_NONE_AUTHENTICATION
+      value: "yes"
+    - name: ETCD_LISTEN_CLIENT_URLS
+      value: "http://0.0.0.0:2379"
+    - name: ETCD_ADVERTISE_CLIENT_URLS
+      value: "http://0.0.0.0:2379"
+    ports:
+    - containerPort: 2379
```

Comment:
I see, so the port you publish here needs to be the same as what the workers connect to above. We should document this so they don't get out of sync.

Reply:
Agree. We should